backend/application/import_observations/parsers/prometheus_trivy/parser.py

```diff
@@ -0,0 +1,167 @@
+import json
+from typing import Optional
+
+import requests
+
+from application.core.models import Observation
+from application.core.types import Severity, Status
+from application.import_observations.models import Api_Configuration
+from application.import_observations.parsers.base_parser import (
+    BaseAPIParser,
+    BaseParser,
+)
+from application.import_observations.types import Parser_Type
+
+STATUS_MAPPING = {
+    "NOT_SET": "",
+    "EXPLOITABLE": Status.STATUS_OPEN,
+    "IN_TRIAGE": Status.STATUS_IN_REVIEW,
+    "RESOLVED": Status.STATUS_RESOLVED,
+    "FALSE_POSITIVE": Status.STATUS_FALSE_POSITIVE,
+    "NOT_AFFECTED": Status.STATUS_NOT_AFFECTED,
+}
+
+
+class PrometheusTrivy(BaseParser, BaseAPIParser):
+    def __init__(self):
+        self.api_configuration: Optional[Api_Configuration] = None
+
+    @classmethod
+    def get_name(cls) -> str:
+        return "Prometheus Trivy"
+
+    @classmethod
+    def get_type(cls) -> str:
+        return Parser_Type.TYPE_OTHER
+
+    def check_connection(
+        self, api_configuration: Api_Configuration
+    ) -> tuple[bool, list[str], dict]:
+        self.api_configuration = api_configuration
+
+        prometheus_trivy_base_url = api_configuration.base_url
+        prometheus_trivy_query = api_configuration.query
+
+        if not prometheus_trivy_base_url.endswith("/"):
+            prometheus_trivy_base_url += "/"
+
+        prometheus_trivy_url = prometheus_trivy_base_url + "api/v1/query?query=" + prometheus_trivy_query
+
+        try:
+            print(prometheus_trivy_base_url)
```

Comment:
There should be no `print`s in the code. Please delete this line.

Reply:
fixed